docroot/modules/custom/civictheme_migrate/src/Plugin/migrate/process/ContentComponents.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Drupal\civictheme_migrate\Plugin\migrate\process;
+
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\civictheme_migrate\ContentComponentFactory;
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\Plugin\MigratePluginManager;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+use Drupal\paragraphs\ParagraphInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Content Components creator.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "ct_content_component_create",
```

Comment:
@govindmaloo 
Please update the name to `civictheme_migrate_content_component_process`